Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Helpers/SmartWeakEvent.cs

```diff
@@ -0,0 +1,174 @@
+﻿// ******************************************************************
```

Comment:
I'm fine with it either way. I could move the smart weak events or remove them all together. Whatever you would like, whatever you think feels best for the project.